provision-proctor/setup.sh

```diff
@@ -81,17 +69,9 @@ if [[ -z "$resourceGroupLocation" ]]; then
     read resourceGroupLocation
 fi
 
-if [[ -z "$proctorName" ]]; then
-    echo "Enter a proctor name to be used to provision proctor resources:"
-    read proctorName
 fi
 
-if [[ -z "$teamName" ]]; then
-    echo "Enter the base team name used for the already provisioned team environments:"
-    read teamName
-fi
-
-if [ -z "$subscriptionId" ] || [ -z "$resourceGroupLocation" ] || [ -z "$proctorName" ] || [ -z "$teamName" ]; then
+if [ -z "$subscriptionId" ] || [ -z "$resourceGroupLocation" ] || [ -z "$azureUserName" ] || [ -z "$azurePassword" ]; then
```

Comment:
This needs `[[ ]]` around each one in order to avoid warnings.

Reply:
Done.